syntaxes/kotlin.tmLanguage.json

```
@@ -19,10 +19,13 @@
     ],
     "repository": {
         "import": {
-            "begin": "\\b(import)\\b\\s*",
+            "begin": "\\b(import)\\b\\s?([\\w+.]*\\w+)?\\s*",
```

Do you have an example of an import with such a modifier? If possible, maybe we could also try keeping the grammar as whitespace-insensitive as possible:

```suggestion
            "begin": "\\b(import)\\b\\s*([\\w+.]*\\w+)?\\s*",
```

I guess the `\b` boundary already takes care of enforcing a space there?